src/Module/Contact.php

```diff
@@ -613,7 +613,7 @@ public static function content(array $parameters = [], $update = 0)
 				'$hidden'         => ['hidden', DI::l10n()->t('Hide this contact from others'), ($contact['hidden'] == 1), DI::l10n()->t('Replies/likes to your public posts <strong>may</strong> still be visible')],
 				'$notify'         => ['notify', DI::l10n()->t('Notification for new posts'), ($contact['notify_new_posts'] == 1), DI::l10n()->t('Send a notification of every new post of this contact')],
 				'$fetch_further_information' => $fetch_further_information,
-				'$ffi_keyword_blacklist' => ['ffi_keyword_blacklist', DI::l10n()->t('Blacklisted keywords'), $contact['ffi_keyword_blacklist'], DI::l10n()->t('Comma separated list of keywords that should not be converted to hashtags, when "Fetch information and keywords" is selected')],
+				'$ffi_keyword_denylist' => ['ffi_keyword_denylist', DI::l10n()->t('Blocklisted keywords'), $contact['ffi_keyword_denylist'], DI::l10n()->t('Comma separated list of keywords that should not be converted to hashtags, when "Fetch information and keywords" is selected')],
```

Comment:
`denylisted` (or whatever is real english)

Reply:
Yup.

Reply:
What about 'blocked' or 'denied' keywords.

Reply:
Done.

Reply:
I uses `Keyword deny list`